modules/nvidia-gpu-aws-enabling-gpu-machines-during-installation.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+//  * installing/installing-nvidia-gpu-aws.adoc
+
+:_content-type: PROCEDURE
+[id="nvidia-gpu-aws-enabling-gpu-machines-during-installation_{context}"]
+= Enabling GPU machines during installation
+
+**Note to reviewers**: This is a very rough first pass at the information based on
+the documentation provided. It needs much more work so please review
+carefully and provide your comments, addition information, Machinesets, etc. and I will schedule a review meeting.
+Much of the information may be redundant and the flow needs work.
+
+**Alternate title**: NVAI-e 2.1 AWS enablement
+
+You can enable machines for NVIDIA AI Enterprise GPU when you are installing OpenShift Container
+Platform on Amazon Web Services (AWS).
+
+OpenShift Container Platform (OCP) uses the Machine API operator to
+fully automate infrastructure provisioning. The Machine API provides full
+stack infrastructure automation on public or private clouds.
+
+You can use the Machine API and the Node Feature Discovery operator
+to define GPU-enabled nodes.
+
+.Prerequisites
+
+* OpenShift CLI
+* OpenShift Container Platform (OCP) installed with https://console.redhat.com/openshift/install/aws/installer-provisioned[Installer Provided Infrastructure (IPI)].
+
+**Note to reviewers**: Are there more prereqs?
+
+The following table lists the validated instance types.
```

Comment:
Maybe we could simply point to NVIDIA's documentation for the supported instance types: https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/platform-support.html#container-platforms

Reply:
@fabiendupont Is there a specfic table on this page you want to point to or just the entire page in generall?